### PR TITLE
don't stop processing workflows from other source when matched

### PR DESCRIPTION
**Pull request by MJedr**
ref: inspirehep/inspirehep#1730

### CODE REVIEW

File: inspirehep/modules/workflows/workflows/article.py [hunk 1]
```diff
@@ -511,9 +500,9 @@
 
         # stop the matched wf and continue this one
         [
-            stop_matched_holdingpen_wfs,
-            mark('stopped-matched-holdingpen-wf', True),
-            save_workflow
+            mark('stopped-matched-holdingpen-wf', False),
+            save_workflow,
+            stop_processing
```
Comment (michamos) [inline inspirehep/modules/workflows/workflows/article.py:505]: I'm confused, why did you change this one?

Comment (MJedr) [reply inspirehep/modules/workflows/workflows/article.py:505]: cause we talked to finish submission workflows, at least it's what I understood.
